packages/css/stories/footnotes/footnotes.stories.ts

```diff
@@ -0,0 +1,22 @@
+import { storiesOfFootnotes, FootnotesTemplateFn } from '@dso-toolkit/stories';
+import { storiesOf } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit-html';
+
+// @ts-ignore
+import readme from './readme.md';
+
+const template: FootnotesTemplateFn<TemplateResult> = () => html`
```

Comment:
In de story definitie uit `@dso-toolkit/stories` maak je wel `FootnotesArgs`. Ook al is deze leeg maak je 'm wel aan (dus goed gedaan) maar je moet 'm wel in de signature van de template functie meenemen: `({}: FootnotesArgs) =>`

Reply:
Check, ik heb hem toegevoegd